src/commands/check.rs

```diff
@@ -182,17 +257,17 @@ pub async fn check(args: CheckArgs) -> anyhow::Result<()> {
 
     if error_count > 0 {
         bail!(
-            "aborting due to previous {error_count} error{s}",
+            "failing due to {error_count} error{s}",
```

Comment:
```suggestion
            "failing with {error_count} error{s}",
```
How do you feel about `with` instead? I'm not certain that's an improvement.

Reply:
Hm. Kinda feels like half-dozen of one, 6 of the other to me. I don't see how it clarifies anything, but it doesn't seem worse in any way. I can make that change if you want. I'm fine with it.